### ISSUE TITLE
数据集与方法

### ISSUE THREAD

**Issue by glorialucheng** [antgroup/Agentar-Scale-SQL#25]
SQL生成模型和SQL选择模型的训练方法以及数据集能否开源一下呢

**Comment by Peng-fei-Wang** [comment]
Hi @glorialucheng ,
Thanks for asking!
1) Data: We used the BIRD training set for the Generator, and the Selector's training data consists of candidates generated from that same set.
2) Training: Both models use the GRPO algorithm.
Detailed hyperparameters and model configurations are provided in the table below:

Category | Hyperparameter | Generator | Selector
-- | -- | -- | --
Data & Model | Base Model | OmniSQL-32B | Qwen2.5-Coder-32B
  | Max Prompt Length | 9216 | 9216
  | Max Response Length | 2048 | 2048
GRPO Algorithm | Reward Function | syntax, execution | selection
  | Mini-Batch Size | 64 | 128
  | Micro-Batch Size / GPU | 1 | 2
  | KL Divergence Coefficient | 0.001 | 0.001
  | Learning Rate | 1e-6 | 1e-6
  | LR Warmup Steps Ratio | 0.1 | 0.1
Rollout | Rollout Engine | SGLang | SGLang
  | GPU Memory Utilization | 0.5 | 0.8
  | N | 16 | 16
  | Log-Prob Micro-Batch Size / GPU | 1 | 2
  | Ref Model Log-Prob Micro-Batch / GPU | 1 | 2
  | Tensor Parallel Size (TP) | 8 | 8
Training | Total Epochs | 3 | 5
  | Total GPUs | 32 | 32
  | GPUs per Node | 8 | 8
  | Total Nodes | 4 | 4
  | GPU Type | A100 | A100
